parallaxinc/Parallax-IDE
# Name update

#### What's this PR do?

Adds images with consistent highlighting and changes project name references to the updated name.
#### Where should the reviewer start?

Take a look at the README
#### How should this be manually tested?

Verify that there are no references to 'ChromeIDE' but 'Parallax IDE'.  Images use the directory/repo name 'Parallax-IDE'
#### Any background context you want to provide?

Updated newly created images to include highlighting.
